scripts/download-binaries.sh

```diff
@@ -67,10 +67,15 @@ curl "${curl_args}O" "${golint_url}" \
     && tar xzfP "${golint_tgz}" -C "${dest_dir}" --strip-components=1 "${golint_dir}/golangci-lint" \
     && rm "${golint_tgz}"
 
+# Install go-bindata tool
+go get -u github.com/go-bindata/go-bindata/...
```

Comment:
A couple of issues:
1. The assumption here is that `GOBIN` is included in the user's `PATH` instead of installing or linking in the kubefed `bin` directory as the other downloads are doing in this script, because the user guide instructs to add kubefed `bin` to the `PATH`. This isn't an issue for me personally as I have `GOBIN` in my `PATH`, but others may run into it.
2. We should probably download and install a known version. If there isn't an official asset we can download, I would suggest checking out a specific tag or commit, that way we're always using the same version.

Reply:
> We should probably download and install a known version. If there isn't an official asset we can download, I would suggest checking out a specific tag or commit, that way we're always using the same version.

That's what I did intially but chaged my approach after getting a comment from @marun  https://github.com/kubernetes-sigs/kubefed/pull/1019#discussion_r314254790

Reply:
@font Let me know your thoughts

Reply:
I think using `go get` is the right thing to do, but we should still use a specific version. Here's an example that was needed with `kind` during a time when there were no releases available. Instead of a commit sha you could use a tag. The only other thing you could add is perhaps a link to it from kubefed's bin directory.
https://github.com/kubernetes-sigs/kubefed/blob/8d0715420cdb4a336a32e5179c3ab1a2024ad309/scripts/download-e2e-binaries.sh#L38-L49